cmd/controller/controller.go

```diff
@@ -230,24 +238,28 @@ func (c *CmdOpts) startController() error {
 			KubeClientFactory: adminClientFactory,
 		})
 	}
-	if !c.SingleNode {
+	if !c.SingleNode && !funk.Contains(c.DisableComponents, "konnectivity-server") {
```

Comment:
Could we use constants for the component names?

Reply:
Probably yes, but looking at this specific file we haven't really used constants earlier. Will try to refactor.

Reply:
Refactored.